### DISCUSSION TITLE
Creating a new branch

### DISCUSSION THREAD

**Discussion by RBTCGP** [community/community#23951] [New to GitHub]
<p>In the docs it says “Click <strong>New Branch</strong>”, but in my repository I do not get this field or menu or option. <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-branches#creating-a-branch" class="inline-onebox" rel="noopener nofollow ugc">Managing branches - GitHub Docs</a></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>It looks like you’re looking at the <a href="https://desktop.github.com/">GitHub Desktop</a> (our desktop GitHub app) documentation. We have documentation for <a href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch">managing branches on the website</a> as well.</p>
